### PR TITLE
Supervisor perspective 9

### LINKED ISSUE #9: Draft immediate supervisor perspective
Add as section to article within same file, similar to #5.

### PR DESCRIPTION
**Pull request by FlicAnderson**
Includes day-to-day supervisor perspective and resolves #9

### CODE REVIEW

File: open-science-projects.md [hunk 1]
```diff
@@ -1,49 +1,102 @@
...
+Completing an open-science project was a very exciting experience, though rather intimidating as it felt like we were contributing to a larger project rather than completing an isolated experiment or literature review. The idea that everything we did could be seen by others and had the potential to help them fuelled this feeling. Knowing that experts within the field could examine the code we had written or critique the conclusions we had reached created an additional layer of motivation in terms of maintaining high quality work that could be understood by others. Every aspect of the project being freely available also removed the anxiety that would have arisen if we had been dealing with confidential ideas or processes, for example testing potential antibiotics. Instead of hiding trade secrets, we were able to freely collaborate and learn from the work of others. Additionally, there was more room for dialogue with other students completing open-science projects. This was a valuable aid as it allowed us to bounce ideas off each other compared to the stress and isolation of a closed project that could not be discussed with peers.
```
Comment (swinterbourne) [inline open-science-projects.md:19]: Are we using "open-science" or "open science" throughout?

File: open-science-projects.md [hunk 2]
```diff
@@ -1,49 +1,102 @@
...
+
+Having two students joining to pursue projects with riboviz at the same time this year was particularly helpful, as a lot of the instruction and coaching were applicable to both, and meant that there was also an opportunity for peer learning. While not all labs might have capacity to offer multiple projects, I was surprised to find that the increase in supervision time required by two students was much less than expected. Many of the same problems were encountered during both projects and could be solved in one discussion.
+
+Supervising two undergraduate projects at the same time was also unexpectedly valuable in highlighting areas where my instruction, knowledge, or communication skills could be improved: _"When one student doesn't seem to understand, it could go either way. When two students don't seem to understand, the problem probably lies with the communicator!"_ The process has been really helpful in making me re-think how I communicate complex ideas, technical information and scientific concepts, and has contributed significantly to my personal development.
```
Comment (swinterbourne) [inline open-science-projects.md:58]: If this is a quote does it need a citation?

File: open-science-projects.md (hunk 1 above)
Comment (ewallace) [reply open-science-projects.md:19]: I vote for "open science", no hyphen. E.G. [Open Life Science](https://openlifesci.org/), [Open Science Framework](https://osf.io).

Comment (swinterbourne) [reply open-science-projects.md:19]: I'll go through and remove any hyphens present

**Comment by ewallace** [comment]
Overall this looks really nice. It tells a coherent narrative, from a different yet mutually supportive perspective from the other bits.

I branched off an issue about organizing the "top takeaways" #13.

Happy to merge if everyone else is?

**Review by 3mma-mack — APPROVED** [review]
I think this reads really well, and is really nice, so I am happy for it to be merged in

**Review by swinterbourne — APPROVED** [review]

File: open-science-projects.md (hunk 2 above)
Comment (FlicAnderson) [reply open-science-projects.md:58]: I'm quoting me, but this is the first time I've said it before, so I'm not sure what style guides recommend for 'quotes of the future'... :grin:

**Pull request merged**